vms/avm/states/state.go

```diff
@@ -275,7 +275,69 @@ func (s *state) DeleteUTXO(utxoID ids.ID) {
 	s.modifiedUTXOs[utxoID] = nil
 }
 
+// TODO: After v1.11.x has activated we can rename [getTx] to [GetTx] and delete
+// [getStatus].
```

Comment:
Q. You mean we will move `getStatus` logic inside `getTx` or just remove the whole `getStatus` calls?

Reply:
After `v1.11.x` we will be guaranteed that everyone will have run `Prune` and that everyone will be running this code (which never writes statuses). So, it will be assumed that any time we currently lookup a status it will return `ErrNotFound`. (So we will completely remove all references to statuses in the state layer)

Reply:
Ah makes sense now. Thx